.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,3 +17,8 @@
 
 ### Changesets
 - [ ] Changes in changelog are generated from PR name. Please, make sure that it explains your changes in an understandable manner. Please, prefix changeset messages with `feat:`, `fix:`, `perf:`, `docs:`, or `chore:`.
+
+---
+
+> [!TIP]
+> The author of this PR can publish a _preview release_ by commenting `/publish` below.
```

Comment:
So can anyone, even non-collaborators, publish `vitest` packages to pkg.pr.new?

Is it tied to Vitest's `secrets.CR_PAT` at least? Meaning that does the service prevents random users from publishing `vitest` and `@vitest/*` packages to the registry?

Reply:
No, only PR author, members and collaborators, that's CR.

Reply:
PR author is "anyone"

Reply:
Looks like the URL actually has the repository owner specified too by default. So no malicious users can publish `https://pkg.pr.new/vitest@<hash>` packages, I think. The `--compact` flag probably does some checks for the origin of the release, @Aslemammad?

Reply:
> PR author is "anyone"

That's CR, we can change it to allow only members to publish via comment or cr-tracked label. There is no npm registry it is just tgz files.